xarray/core/dataset.py

```diff
@@ -792,13 +806,19 @@ def chunks(self):
         array.
         """
         chunks = {}
-        for v in self.variables.values():
+        for v in self.data_vars.values():
```

Comment:
I am concerned about skipping non-`data_vars` here. Coordinates could still be chunked, e.g., if they were loaded from a file, or created directly from dask arrays.